src/subsystems/nodejs/builders/granular/devShell.nix

```diff
@@ -39,8 +39,6 @@ mkShell {
     nodeModulesDrv = pkg.overrideAttrs (old: {
       buildPhase = ":";
       installMethod = "copy";
-      src = "";
-      dontUnpack = true;
```

Comment:
We need to read `package.json` to see what the direct dependencies are, to give priority to linking their bins.